nuxeo-coldstorage-package/src/main/resources/install/templates/coldstorage/nuxeo.defaults

```diff
@@ -3,6 +3,9 @@ nuxeo.template.includes=s3binaries
 #GLACIER
 coldstorage.target=.
 
+# Every hour at the 7th minute
+nuxeo.coldstorage.check.retrieve.state.cronExpression=0 7 * ? * * *
+
```

Comment:
should be removed, right? as for now we didn't add the configurable way for the scheduler cron.

Reply:
Hello
if your question refers to the unit tests, we have this [new file ](https://github.com/nuxeo/nuxeo-coldstorage/blob/feature-NXP-30343-restore-glacier-document/nuxeo-coldstorage/src/test/resources/OSGI-INF/coldstorage-scheduler-contrib-test-disable.xml) to disable the scheduler cron.

Reply:
cool sorry ;)